Doc/Tutorial/TutorialBasicMonomorphic.lhs

```diff
@@ -67,12 +67,6 @@ manipulation tutorial you can see an example of when they might differ.
 >                                       , tableField "age"
 >                                       , tableField "address" ))
 
-> personTable' :: Table (Field SqlText, Field SqlInt4, Field SqlText)
```

Comment:
This definition is not used anywhere, so just deleting it.

Reply:
I suspect it was supposed to be an example that used `tableWithSchema`. If you'd like to change it to that then that would be helpful, otherwise I'll do it myself when I get round to it.

Reply:
Sure, I changed it to tableWithSchema.